hydra-cardano-api/src/Hydra/Cardano/Api/UsingRawBytesHex.hs

```diff
@@ -3,15 +3,15 @@
 -- With some minor modifications of also using encodeUtf8 in 'IsString' instance.
 module Hydra.Cardano.Api.UsingRawBytesHex where
 
-import Hydra.Cardano.Api.Prelude
+import Hydra.Cardano.Api.Prelude hiding (Show, error, show)
 
 import Data.Aeson (FromJSONKey, ToJSONKey)
 import qualified Data.Aeson as Aeson
 import qualified Data.Aeson.Types as Aeson
 import qualified Data.ByteString.Base16 as Base16
-import Data.String (IsString (..))
 import qualified Data.Text as Text
 import Data.Typeable (tyConName, typeRep, typeRepTyCon)
+import Prelude (Show, error, show)
```

Comment:
Why is the `Hydra.Cardano.Api.Prelude` not already exporting the right `show` and `error`? I saw this override in at least two modules.

Should: Do re-export what you need in `Hydra.Cardano.Api.Prelude` and don't mix it with the base `Prelude`.

Reply:
It is a bit of a mix where some modules use the base `Prelude` while others use our own `Hydra.Cardano.Api.Prelude`. I reverted the changes in our custom Prelude so these changes are not needed.